resources/inviwo.qrc

```diff
@@ -100,6 +100,7 @@
         <file>svgicons/file-warning.svg</file>
 
         <file>svgicons/haptics.svg</file>
+        <file>svgicons/trashcan.svg</file>
```

Comment:
should this not be in the animationqt resources?

Reply:
True, did not know that it had its own